lib/plugins.js

```diff
@@ -265,6 +265,10 @@ exports.categories = {
             url: 'https://github.com/felixheck/bissle',
             description: 'Minimalist HALicious pagination reply interface for HapiJS'
         },
+        cron: {
+          url: 'https://github.com/antonsamper/hapi-cron',
```

Comment:
Can you please fix indentation here (Expected indentation of 12 space characters but found 10  indent)